Index.html

```diff
@@ -13,15 +13,28 @@
 </head>
 <body>
     <header>
-        <span><a href="#"><b>My Logo</b></a></span>
-        <nav id="switch">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
+      <a href="#" id="Logo"><b>David Katende</b>
```

Comment:
- [OPTIONAL] Excellent job for hiding the nav-links in mobile version 🥇 I think it would be better to align logo and menu as given in the figma design guidelines 💪I highly recommend fixing this. See the images below for more clarity 💡

Your design before 🔽 | Your design now 🔽 | Figma guidelines 🔽|
---| ---| ---|
![image](https://user-images.githubusercontent.com/106119105/230498244-2e2c85d0-7e40-4ba6-a7e1-88a0ffe5a4fa.png) | ![image](https://user-images.githubusercontent.com/106119105/230498337-5196a0d1-94cb-49c2-89bf-e0d83e693538.png) | ![image](https://user-images.githubusercontent.com/106119105/230498244-2e2c85d0-7e40-4ba6-a7e1-88a0ffe5a4fa.png) |